components/status/StatusPoll.vue

```diff
@@ -27,13 +27,18 @@ async function vote(e: Event) {
   }
   poll.voted = true
   poll.votesCount++
-  poll.votersCount = (poll.votersCount || 0) + 1
+
+  if (!poll.votersCount && poll.votesCount)
```

Comment:
`poll.votesCount` is always a number, not including `null`

Reply:
Is there a change you would suggest here?

Reply:
Relavent to this? https://elk.zone/m.webtoo.ls/@cheeaun@mastodon.social/109688468147731159

Reply:
I think this PR is good to go. It doesn't matter if votersCount (or votesCount) is null or 0 - we fall back to zero in that case.